src/main/scala/redis/Redis.scala

```diff
@@ -23,13 +23,13 @@ trait RedisCommands
   with Server
   with HyperLogLog
 
-abstract class RedisClientActorLike(system: ActorRefFactory) extends ActorRequest {
+abstract class RedisClientActorLike(system: ActorSystem) extends ActorRequest {
```

Comment:
you should have something like that 
`(system: ActorRefFactory, dispatcher : MessageDispatcher)